### PR TITLE
Prevent user id field from being dragged into field group

### PR DESCRIPTION
**Pull request by AbdiTolesa**
Fix https://github.com/Strategy11/formidable-pro/issues/5225

CleanShot 2024-08-01 at 22 36 38

### CODE REVIEW

File: js/formidable_admin.js [hunk 1]
```diff
@@ -788,7 +788,12 @@ function frmAdminBuildJS() {
 		document.querySelectorAll( sortableSelector ).forEach(
 			list => {
 				makeDroppable( list );
-				Array.from( list.children ).forEach( child => makeDraggable( child, '.frm-move' ) );
+				Array.from( list.children ).forEach( child => {
```
Comment (Crabcyborg) [inline js/formidable_admin.js:791]: @AbdiTolesa I don't think you'll want to update the initialization code to fix this.

Your update should be contained to the `allowDrop` function.

Comment (Crabcyborg) [reply js/formidable_admin.js:791]: Going further, you should really only need to update `allowNewFieldDrop`.

Comment (AbdiTolesa) [reply js/formidable_admin.js:791]: @Crabcyborg Yeah I was just updating that already but just saw your comment now.

**Review by Crabcyborg — APPROVED** [review]
Thanks for working on this @AbdiTolesa.

I moved some code around and fixed an issue where you could still drag fields beside a user ID field in a field group that way.

The user ID check makes more sense where the hidden field check is, and the function `groupIncludesBreakOrHidden` needed to also start checking for user id fields since that helper function covers the similar issue with hidden fields.

I think this is good to go now.

🚀

**Comment by AbdiTolesa** [comment]
> Thanks for working on this @AbdiTolesa.
> 
> I moved some code around and fixed an issue where you could still drag fields beside a user ID field in a field group that way.
> 
> The user ID check makes more sense where the hidden field check is, and the function `groupIncludesBreakOrHidden` needed to also start checking for user id fields since that helper function covers the similar issue with hidden fields.
> 
> I think this is good to go now.
> 
> 🚀

Thanks @Crabcyborg!

**Pull request merged**